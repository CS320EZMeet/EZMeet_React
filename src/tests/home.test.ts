import renderer from 'react-test-renderer';
import Home from '../pages/home';

it('renders home correctly', () => {
    const tree = renderer
        .create( Home() )
        .toJSON();
    expect(tree).toMatchSnapshot();
});