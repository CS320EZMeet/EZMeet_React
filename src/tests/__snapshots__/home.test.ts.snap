// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders home correctly 1`] = `
<div>
  <h1>
    Welcome to EzMeet
  </h1>
</div>
`;
